lib/rendition.js

```diff
@@ -279,6 +282,26 @@ class Rendition {
         return obj;
     }
 
+    embed() {
+        return Number.isInteger(this.instructions.embedBinaryLimit)
+            && this.instructions.embedBinaryLimit <= EMBED_LIMIT_MAX
+            && this.size() <= this.instructions.embedBinaryLimit;
```

Comment:
Try to add a unit test which embeds something that has the size limit? Wdyt?

Reply:
(You can generate a random array buffer with a given size - the unit test I mention doesn't really care about the data. I just want to have a test to verify the size limited is included and will upload properly!)

Reply:
I have a test where the `rendition.size()` > `this.instructions.embedBinaryLimit`: https://github.com/adobe/asset-compute-sdk/blob/d1cf3f23e7a00c11eca8927d45f4ba9b9dbd8321/test/api.test.js#L751 (by making the embedBinaryLimit super small)

and a test inside rendition tests where the `rendition.size()` > `this.instructions.embedBinaryLimit`: https://github.com/adobe/asset-compute-sdk/blob/d1cf3f23e7a00c11eca8927d45f4ba9b9dbd8321/test/rendition.test.js#L303 (by mocking the size of the rendition to be very large)

could you explain more what you mean by `something that has the size limit`? (size limit depends on user parameter `embedBinaryLimit` so my tests already cover this). I chose not to do a test where the rendition size is actually very large (maybe even the limit 32kb) because that would slow down testing

Reply:
I'm looking for a rendition of size 32kb indeed. How come this would slow down testing? 32kb is still quite small... ?

Reply:
sorry I was confusing this with the data uri limit